recipes/microservice-essentials/all/conanfile.py

```diff
@@ -0,0 +1,104 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain
+from conan.tools.files import get, copy
+from conan.tools.build import check_min_cppstd
+import os
+
+required_conan_version = ">=1.53.0"
+
+class MicroserviceEssentials(ConanFile):
+    name = "microservice-essentials"    
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/seboste/microservice-essentials"
+    license = "MIT"    
+    description = """microservice-essentials is a portable, independent C++ library that takes care of typical recurring concerns that occur in microservice development."""
+    topics = ("microservices", "cloud-native", "request-handling")
+    settings = "os", "compiler", "arch", "build_type"    
+    package_type = "library"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_tests": [True, False],
+        "with_examples": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_tests": False,
+        "with_examples": False
+    }
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "7",
+            "clang": "5",
+            "apple-clang": "10",
+            "Visual Studio": "15.7",
```

Comment:
Missing `msvc` compiler version check :)

```suggestion
            "Visual Studio": "15.7",
            "msvc": "191",
```